server/aws-lsp-codewhisperer/src/language-server/codeWhispererSecurityScanServer.ts

```diff
@@ -47,21 +47,25 @@ export const SecurityScanServerToken =
             }
             try {
                 if (!credentialsProvider.hasCredentials('bearer')) {
-                    throw new Error('credentialsProvider does not have bearer token credentials')
+                    throw new Error('Credentials provider does not have bearer token credentials')
                 }
                 if (!params.arguments || params.arguments.length === 0) {
-                    throw new Error(`Incorrect params provided. Params: ${params}`)
+                    throw new Error(`Error: Incorrect parameters parameters provided`)
```

Comment:
Since the error text becomes user-facing, lets log (with `logging`) this error message, along with `${params}`. This will let us diagnose the problem and see what the input is.

Meanwhile, we can make this error text less "technical" by saying something like "Error: Invalid data"